src/libdiod/diod_log.c

```diff
@@ -26,15 +26,27 @@
 
 #include "diod_log.h"
 
-static char *prog = NULL;
-
+static char log_prefix[32] = { 0 };
```

Comment:
commit message nit: "some TAP unit test" -> "some TAP unit tests"

Reply:
Thanks!  Fixed and I'll set MWP.